src/app/(pages)/users/_components/UsersSearch.tsx

```diff
@@ -26,11 +23,13 @@ const UsersSearch: React.FC<Props> = ({ onSearch }) => {
         onSearch={handleSearch}
         searchKey={"lastname"}
       />
-      <Dropdown
-        cityNames={cityNames}
-        onSearch={handleSearch}
-        searchKey={"city"}
-      />
+      {cityNames?.length && (
+        <Dropdown
+          values={cityNames}
```

Comment:
The values to this dropdown should be adjusted so they match the new suggested type `DropdownOption`.